scripts/monitoring/cron-send-user-pod-count.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+if [ $# -eq 1 ]; then
+  if [[ "RUNNING" = $1 ]]; then
+      COUNT_RUNNING=$(KUBECONFIG=/etc/openshift/master/admin.kubeconfig oadm manage-node --list-pods --selector=type=compute | grep Running | wc -l  ; exit ${PIPESTATUS[0]})
+      echo
+      echo "Number of User Running pod account : $COUNT_RUNNING"
+      echo
+      echo "Running: ops-zagg-client -k 'openshift.master.user.pod.running.count' -o '$COUNT_RUNNING'"
+      ops-zagg-client -k "openshift.master.user.pod.running.count" -o "$COUNT_RUNNING"
+      echo
+  elif [[ "TOTAL" = $1 ]]; then
+      COUNT_TOTAL=$(KUBECONFIG=/etc/openshift/master/admin.kubeconfig oadm manage-node --list-pods --selector=type=compute  | wc -l  ; exit ${PIPESTATUS[0]})
+      echo
+      echo "Number of User pod Total account : $COUNT_TOTAL"
+      echo
+      echo "Running: ops-zagg-client -k 'openshift.master.pod.total.count' -o '$COUNT_TOTAL'"
+      ops-zagg-client -k "openshift.master.pod.total.count" -o "$COUNT_TOTAL"
+      echo 
+  else
+      echo "Only RUNNING or TOTAL is acceptable"
+  fi
+
+else
+  echo "Usage:`basename $0` RUNNING|TOTAL"
+fi
```

Comment:
Why not just always send both running and total pod counts to zabbix?

It's fine to send multiple values in the same script. In fact, at some point, we're going to combine a bunch of these together.